azure-pipelines-integration.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 - job: VS_Integration
   pool:
     name: NetCorePublic-Pool
-    queue: buildpool.windows.10.amd64.vs2019.pre.open
+    queue: buildpool.windows.10.amd64.vs2019.open
```

Comment:
So the reason for changing this is because when this change is made vs2019.pre is 16.7 and now vs2019 is 16.7?

Reply:
Correct. We do not want to merge any of this change back into master. 16.8 already has the right things for this. We do want to test this on 16.7, which is now the released version, and integration tests won't pass on 16.8